sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -14,11 +14,52 @@ static struct snd_soc_acpi_codecs tgl_codecs = {
 	.codecs = {"MX98357A"}
 };
 
+static const u64 rt711_0_adr[] = {
+	0x000010025D071100
+};
+
+static const u64 rt1308_1_1_adr[] = {
+	0x000110025D130800
+};
+
+static const u64 rt1308_1_2_adr[] = {
+	0x000112025D130800
+};
+
```

Comment:
intel-sdw sdw-master-1: SDW Slave Addr: 22025d130800
 intel-sdw sdw-master-1: SDW Slave class_id 0, part_id 1308, mfg_id 25d, unique_id 2, version 2
 intel-sdw sdw-master-1: SDW Slave Addr: 20025d130800
 intel-sdw sdw-master-1: SDW Slave class_id 0, part_id 1308, mfg_id 25d, unique_id 0, version 2

kernel log for TGL

Reply:
updated, thanks.